frontend/packages/kubevirt-plugin/integration-tests/views/virtualMachine.view.ts

```diff
@@ -73,6 +73,8 @@ export const vmDetailstatusButton = (namespace, name) =>
 
 export const vmDetailService = (serviceName) => $(`[data-test-id="${serviceName}"]`);
 
+export const vmLoggedInUsers = element(by.xpath('//a[contains(@href,"logged-in-users")]'));
```

Comment:
Can we use css selector instead of xpath, seems css is recommended over xpath.

https://stackoverflow.com/questions/31075513/which-one-of-the-locator-is-efficient-by-css-or-by-xpath-or-by-id-in-protractor

Reply:
we can, once the developer add a css selector - we can change the test